### DISCUSSION TITLE
Why prefetching is happening without a hover?

### DISCUSSION THREAD

**Discussion by 1alexvash** [vercel/next.js#40449] [Help] [answered]
In this example NextJS is prefetching the content without it being in the visible area of the screen before any hover
![animation](https://user-images.githubusercontent.com/46556479/189546312-fd7bb5f4-3331-417d-8f74-0ef962876c23.gif)

Can someone educate me on why it is happening?

**Comment by gaston-flores** [comment, marked as answer]
See the [next/link documentation](https://nextjs.org/docs/api-reference/next/link):

> `prefetch` - Prefetch the page in the background. Defaults to true. Any <Link /> that is in the viewport (initially or through scroll) will be preloaded. Prefetch can be disabled by passing prefetch={false}. When prefetch is set to false, prefetching will still occur on hover. Pages using [Static Generation](https://nextjs.org/docs/basic-features/data-fetching/get-static-props) will preload JSON files with the data for faster page transitions. Prefetching is only enabled in production.

If you want to prefetch only on hover, set `prefetch` to false in your `Link`
